fouleggs/src/de/up/hpi/informationsystems/fouleggs/dactors/AdminSession.scala

```diff
@@ -43,7 +43,8 @@ class AdminSession extends Actor {
   }
 
   def addCastToFilm(personId: Int, filmId: Int, roleName: String): Unit = {
-    context.system.actorOf(Props(new CastAndFilmographyFunctor(personId, filmId, roleName, self)))
+    val functor: ActorRef = context.system.actorOf(Props(new CastAndFilmographyFunctor(personId, filmId, roleName, self)))
```

Comment:
```scala
Props(new SomeActor(params*))
```
This is really not recommended to use in another actor. Please see [Akka Documentation - Props](https://doc.akka.io/docs/akka/2.5.5/scala/actors.html) for the recommended way to create `Props` instances, which I really would argue for: Define them in the companion object:
```scala
object CastAndFilmographyFunctor {
  def props(pId, fId, role, caller) = Props(new CastAndFilmographyFunctor(...))
}
```